certified-connectors/Redque/Intro.md

```diff
@@ -0,0 +1,20 @@
+## Prerequisites
```

Comment:
@jirisodomka thank you so much for your submission. May I ask you please to remove "Intro.md" and "redque.png" as they are not required in the open source repository.

Reply:
Hello @Amjed-Ayoub thank you for the review. I will do necessary changes you mentioned.